SPECS/LICENSES-AND-NOTICES/data/licenses.json

```diff
@@ -752,6 +760,13 @@
             "specs": [
                 "ceph"
             ]
+        },
+        {
+            "origin": "Vincent Batts Open Source",
+            "license": "????",
```

Comment:
This license entry seems odd. Let's check with @PawelWMS on the best way to deal with this and annotate the reasoning in the PR description for the future.